Assets/Scripts/Augment/AugmentImplementations/DDRBody.cs

```diff
@@ -50,7 +50,10 @@ public class DDRBody : GunBody
     [SerializeField]
     private Precision[] precisionsGoodToBad;
 
-    LTDescr arrowMover;
+    int arrowMover;
+    int screenFlasher;
+    int screenPulseAnimator;
+    int textAnimator;
```

Comment:
These names are inconsistent with the `Tween` suffix that you've used elsewhere, perhaps rename these to match that?

Reply:
Fixed 👍